ospfd/ospf_lsa.c

```diff
@@ -2210,7 +2210,7 @@ struct ospf_lsa *ospf_external_lsa_originate(struct ospf *ospf,
 		if (IS_DEBUG_OSPF_EVENT)
 			zlog_debug(
 				"LSA[Type5:%pI4]: deferring AS-external-LSA origination, router ID is zero",
-				&ei->p.prefix);
+				ei ? &ei->p.prefix : NULL);
```

Comment:
Wont you still get the crash here since you are trying to access NULL address for printing the log?

Reply:
printfrr_i4 is fine with null.

Reply:
```(gdb) 
467		zlog_debug("HI HI HI %pI4", NULL);
(gdb) 
468		frr_run(zrouter.master);
(gdb) n
2022/02/18 10:14:41 ZEBRA: [JFNKF-0M9BM] HI HI HI (null)
```